.github/workflows/main.yml

```diff
@@ -0,0 +1,82 @@
+name: Build and Test
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      FC: gfortran-9
+      CC: gcc-9
+      CXX: g++-9
+
+    steps:
+
+    - name: install-dependencies
+      run: |
+        sudo apt-get install libpng-dev zlib1g-dev libjpeg-dev
+
+    - name: checkout-jasper
+      uses: actions/checkout@v2
+      with:
+        repository: mdadams/jasper
+        path: jasper
+        ref: version-2.0.16
+
+    - name: cache-jasper
+      id: cache-jasper
+      uses: actions/cache@v2
+      with:
+        path: ~/jasper
+        key: jasper-${{ runner.os }}-${{ hashFiles('jasper/VERSION') }}
+
+    - name: build-jasper
+      if: steps.cache-jasper.outputs.cache-hit != 'true'
+      run: |
+        cd jasper
+        mkdir cmake_build
+        cd build
+        cmake .. -DCMAKE_INSTALL_PREFIX=~/Jasper
+        make -j2
+        make install
```

Comment:
any reason why Jasper cannot be treated as png, zlib and jpeg?

Reply:
Is there a particular minimum version of Jasper that is necessary? If so, that should be documented.

Reply:
zlib and png are installed by default on Github runners (and maybe Ubuntu as a whole), and are available as Ubuntu packages, Jasper isn't.

As far as version goes, I don't know. I've seen 1.9.0, 1.9.28, and 2.xx. I just picked the version that I found in NCEPLIBS-externals.